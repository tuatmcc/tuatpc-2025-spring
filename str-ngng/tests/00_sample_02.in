41 5
tokyouniversityofagricultureandtechnology
6 28
2 22
20 36
1 35
1 41
