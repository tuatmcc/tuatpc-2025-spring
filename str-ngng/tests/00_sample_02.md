$S$ は英小文字のみからなる文字列であることが保証されています。