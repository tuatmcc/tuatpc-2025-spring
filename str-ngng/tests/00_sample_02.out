20
13
7
123
245
